### DISCUSSION TITLE
Bed Clips ( MESH_INSET %variable%) adjustment in advanced?

### DISCUSSION THREAD

**Discussion by kentsangcanada** [Jyers/Marlin#309] [Q&A] [answered]
Bed Clips ( MESH_INSET %variable%) adjustment in advanced?

Any way we can add Mesh Inset to the advanced menu to set the Mesh Bounds so we dont hit our bed clips?

![Capture](https://user-images.githubusercontent.com/6929546/112045176-ec9f2880-8b20-11eb-9439-71a2930b7cf6.JPG)

**Comment by Jyers** [comment, marked as answer]
I just took a look and unfortunately it seems mesh inset can only be set at compile time. If it is a prevalent enough issue, I may increase the default inset a bit.

**Reply by kentsangcanada** [reply]
Thank you for Trying I mean I dont ever print that close, but it just looks ODD

**Reply by samsp-msft** [reply]
I also have this problem, that it can hit a clip when measuring. Is there something better than binder clips to hold the glass bed firmly to the base plate?

**Reply by Jyers** [reply]
I believe that binder clips are the usual choice. I now realized that the probing margin needs to be increased anyway, so that should solve the issue.
